Encounter Me/Modal.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+public class Modal
+{
+	public string modalDisplay { get; set; }
+	public string modalClass { get; set; }
```

Comment:
Sorry for nit-picking but I think this does not conform with naming convections in C# - public properties names should start with a capital letter.